Signal/src/AppDelegate.m

```diff
@@ -517,6 +517,9 @@ - (void)applicationDidBecomeActive:(UIApplication *)application {
                 // This will fetch new messages, if we're using domain
                 // fronting.
                 [[PushManager sharedManager] applicationDidBecomeActive];
+
+                // If there were any messages in our local queue which we hadn't yet processed.
+                [[OWSMessageReceiver sharedInstance] handleAnyUnprocessedEnvelopes];
```

Comment:
Why not have `OWSMessageReceiver` observe `UIApplicationDidBecomeActiveNotification`?  I prefer that approach so that everything is independent as possible.